elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -272,9 +281,28 @@
          */
         onModelValueChange: function(newVal){
             this._debug( "urth-core-dataframe onModelValueChange", newVal );
+            newVal = this._deserializeColumnTypes(newVal);
             this._setValue(newVal || {data:[]});
         },
 
+        /*
+         * Returns the deserialized DataFrame reconstructed via the column_types
+         * @param df - the serialized DataFrame
+         * @return the reconstructed DataFrame
+         */
+        _deserializeColumnTypes: function(df) {
+            for (var i=0; i<df.column_types.length; i++) {
+                //if this column is of type date convert the string content to a date object
+                if(df.column_types[i] === "datetime64[ns]" || df.column_types[i] === "Date"
```

Comment:
yup, sorry, didn't see your comment.  However, we probably should keep track of some of the subtle differences between these types, such as precision (at least date vs. datetime?)  I also thought there was a concept of time zone vs local dates in at least some of the kernels.